src/assets/index.ts

```
@@ -151,6 +151,13 @@ export { default as LogoMedium } from './svg/logo-medium.svg';
 export { default as DefaultPerson } from './svg/default-person.svg';
 export { default as MobileDefaultPerson } from './svg/mobile-default-person.svg';
 export { default as MobilePlus } from './svg/mobile-plus.svg';
+export { default as MobileComment } from './svg/mobile-comment.svg';
+export { default as CryEmoji } from './svg/cry-emoji.svg';
+export { default as MobileLikeButtonDF } from './svg/mobile-like-button-df.svg';
+export { default as MobileLikeButtonPR } from './svg/mobile-like-button-pr.svg';
+export { default as MobileArrowUp } from './svg/mobile-arrow-up.svg';
+export { default as MobileArrowDown } from './svg/mobile-arrow-down.svg';
+export { default as MobileReport } from './svg/mobile-report.svg';
```

ESLint

[error] Multiple exports of name 'MobileReport'.

(import/export)